django/contrib/admin/options.py

```diff
@@ -829,6 +829,15 @@ def get_action(self, action):
             description = capfirst(action.replace('_', ' '))
         return func, action, description
 
+    def get_empty_value_display(self):
```

Comment:
It might make sense to give this method an optional `field` or `field_name` argument. That way a user can decide based on the field(name) what the empty value should be.